dask-gateway-server/dask-gateway-proxy/pkg/sni/sni.go

```diff
@@ -31,7 +31,7 @@ func (c *peekedTCPConn) Read(p []byte) (n int, err error) {
 	return c.TCPConn.Read(p)
 }
 
-func wrapPeeked(inConn *net.TCPConn, br *bufio.Reader) tcpConn {
+func wrapPeeked(inConn *net.TCPConn, br *bufio.Reader) TcpConn {
```

Comment:
Is it correct that you adopt MixedCase, a combination of camelCase and PascalCase, where...

> If an identifier needs to be visible outside the package, its first character should be uppercase.

?

Reply:
Correct. See [go spec docs on Exported Identifiers](https://go.dev/ref/spec#Exported_identifiers) for reference. 
>An identifier may be exported to permit access to it from another package. An identifier is exported if both: 
>1. the first character of the identifier's name is a Unicode upper case letter (Unicode class "Lu"); and

>2. the identifier is declared in the [package block](https://go.dev/ref/spec#Blocks) or it is a [field name](https://go.dev/ref/spec#Struct_types) or [method name](https://go.dev/ref/spec#MethodName).

Reply:
Thanks for helping me learn about this @rigzba21!